.eslintrc

```diff
@@ -1,7 +1,18 @@
+extends:
+  - plugin:fxa/server
+  - plugin:node/recommended
```

Comment:
@deeptibaghel we would need to add this here: https://github.com/mozilla/eslint-plugin-fxa/issues/9 then `plugin:fxa/server` should have the new rules automatically and can be reused in other repositories.